script.js

```diff
@@ -113,11 +118,16 @@ function initializeLibrary(data) {
     tabsContainer.innerHTML = '';
     tabContentsContainer.innerHTML = '';
 
-    // Tab for entries with read history but not in library
-    categories.unshift({ name: 'History', order: 65535 }); // Sets history tab's order as last
+    // Add 'History' tab if it doesn't exist
+    if (!categories.some(category => category.name === 'History')) {
+        categories.push({ name: 'History', order: 65535 }); // Sets history tab's order as last
+    }
+
+    // Ensure 'Default' tab is always first, but add only if it doesn't exist
+    if (!categories.some(category => category.name === 'Default')) {
+        categories.unshift({ name: 'Default', order: -1 });
+    }
 
-    // Ensure 'Default' tab is always first
-    categories.unshift({ name: 'Default', order: -1 });
 
```

Comment:
Added a check to see if `Default` and `History` already exists, because I had an issue where if I refresh (change sort order) the library, it would duplicate them